libs/types/src/ballot_package.ts

```diff
@@ -5,11 +5,28 @@ import {
   PrecinctId,
 } from './election';
 import { SystemSettings } from './system_settings';
+import { BallotPackageMetadata } from './ballot_package_metadata';
+import { UiStringAudioClips } from './ui_string_audio_clips';
+import { UiStringAudioKeysPackage } from './ui_string_audio_keys';
+import { UiStringsPackage } from './ui_string_translations';
+
+export enum BallotPackageFileName {
+  APP_STRINGS = 'appStrings.json',
+  AUDIO_CLIPS = 'audioClips.jsonl',
+  AUDIO_KEYS = 'audioKeys.json',
+  ELECTION = 'election.json',
+  METADATA = 'metadata.json',
```

Comment:
Putting the ballot package version in a separate `metadata.json` file, as opposed to the `systemSettings.json` makes sense to me. The ballot package version feels higher-level than any one file within the ballot package (and could even dictate which files are present to begin with, somewhere down the road)